tests/unit/createWidgetBase.ts

```diff
@@ -255,6 +255,26 @@ registerSuite({
 			assert.isUndefined(result.children);
 			assert.equal(result.text, 'I am a text node');
 		},
+		'instance gets passed to VNodeProperties as bind to widget and all children'() {
```

Comment:
we should also probably test that `bind` can still be used as usual (overridden)